code/modules/research/research_disk.dm

```diff
@@ -20,3 +20,57 @@
 /obj/item/disk/tech_disk/debug/Initialize()
 	. = ..()
 	stored_research = new /datum/techweb/admin
+
+/obj/item/research_notes
+	name = "research notes"
+	desc = "Valuable scientific data. Use it in a research console to scan it."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "paper"
+	item_state = "paper"
+	///research points it holds
+	var/value = 69
+	///origin of the research
+	var/typee = "debug"
+	///if it ws merged with different origins to apply a bonus
+	var/mixed = FALSE
+
+/obj/item/research_notes/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>It is worth [value] research points.</span>"
+
+/// proc that changes name and icon depending on value
+/obj/item/research_notes/proc/change_vol()
+	if(value >= 10000)
+		name = "revolutionary discovery in the field of [typee]"
+		icon_state = "docs_verified"
+		return
+	else if(value >= 2500)
+		name = "essay about [typee]"
+		icon_state = "paper_words"
+		return
+	else if(value >= 100)
+		name = "notes of [typee]"
+		icon_state = "paperslip_words"
+		return
+	else
+		name = "fragmentary data of [typee]"
+		icon_state = "scrap"
+		return
+
+///proc when you slap research notes into another one, it applies a bonus if they are of different origin (only applied once)
+/obj/item/research_notes/proc/merge(obj/item/research_notes/new_paper)
+	value = value + new_paper.value
+	if(typee != new_paper.typee && !mixed)
+		value = value * 1.3
+		typee = "[typee] and [new_paper.typee]"

```

Comment:
It's absolutely ok to use a list for this behaviour. That way it can be more than one. Maybe even just make it display "various fields" should it be more than three or something.

Reply:
cobby said "departmental interaction" somewhere

Reply:
kinda off-topic but use 
`value *= 1.3`
looks alot more professional

Reply:
I mean it's fine but put it in the description somewhere.

Reply:
hmmm let's not do this


Source 1 | Source 2 | Total Value
-- | -- | --
1000 | 0 | 1000
1000 | 1 | 1301.3